addons/web/static/src/core/overlay/overlay_service.js

```diff
@@ -46,6 +46,7 @@ export const overlayService = {
                 props,
                 remove: removeCurrentOverlay,
                 sequence: options.sequence ?? 50,
+                rootId: options.rootId,
```

Comment:
Don't forget to adapt the typedef `OverlayServiceAddOptions` to document new option `rootId`